src/main/scala/ai/privado/passes/PropertyParserPass.scala

```diff
@@ -312,7 +312,13 @@ class PropertyParserPass(cpg: Cpg, projectRoot: String, ruleCache: RuleCache, la
 
     SourceFiles
       .determine(Set(projectRoot), extensions)
-      .concat(getListOfFiles(projectRoot).map(f => f.getAbsolutePath).filter(_.matches(".*\\.env.*")))
+      .concat(
+        getListOfFiles(projectRoot)
+          .map(f => {
+            f.getAbsolutePath
+          })
+          .filter(_.matches(".*\\.env(?!.*(?:.js|.py|.java|.sh)$).*"))
```

Comment:
Add .json, .ts also in filter

Reply:
We are scanning .json files separately. Will add .ts